src/main/java/gov/nist/secauto/oscal/lib/profile/resolver/support/BasicIndexer.java

```diff
@@ -190,11 +190,14 @@ public Collection<IEntityItem> getEntitiesByItemType(@NonNull IEntityItem.ItemTy
     return entityGroup == null ? CollectionUtil.emptyList() : ObjectUtils.notNull(entityGroup.values());
   }
   //
-  // public EntityItem getEntity(@NonNull ItemType itemType, @NonNull UUID identifier) {
-  // return getEntity(itemType, ObjectUtils.notNull(identifier.toString()), false);
+  // public EntityItem getEntity(@NonNull ItemType itemType, @NonNull UUID
```

Comment:
Apologies, I worked in this file and auto-formatting with project rules triggered this. Revert?

Reply:
Woops, disregard.